pkgs/development/libraries/qt-6/modules/qtmqtt.nix

```diff
@@ -0,0 +1,20 @@
+{ qtModule
+, stdenv
+, lib
+, fetchurl
+, qtbase
+, udev
+, pkg-config
+}:
+
+qtModule rec {
+  pname = "qtmqtt";
+  version = "6.4.3";
+  src = fetchurl {
+    url = "https://github.com/qt/qtmqtt/archive/refs/tags/v${version}.tar.gz";
+    sha256 = "P2CpIHVx0ya4eJf/UYFEitJEdCa/Qn7PD1CUCmY4TrE=";
+    name = "v${version}.tar.gz";
```

Comment:
Isn't this the default?

Reply:
I copied it over when I moved it from `srcs` without thinking... yes it is not necessary.